Steer/DigitizerWorkflow/src/TPCDigitizerSpec.cxx

```diff
@@ -141,13 +141,18 @@ class TPCDPLDigitizerTask
     if (!gGeoManager) {
       o2::base::GeometryManager::loadGeometry(o2::conf::DigiParams::Instance().digitizationgeometry);
     }
+
+    if (!mWriteGRP) {
+      // the GRP data is only sent from one of the parallel processors. If pipelines are created
+      // using parallelPipeline method, the mWriteGRP flag can not be set and in this case we
+      // get the condition from the parallel context.
+      auto const& parallelContext = ic.services().get<ParallelContext>();
+      mWriteGRP = parallelContext.index1DSize() > 1 && parallelContext.index1D() == 0;
```

Comment:
why do you need to check ``index1DSize() > 1`` ? Would not this prevent from writing the GRP in case just 1 lane was requested?

Reply:
good catch, thanks!

Reply:
Could not implement the logic as needed with the ParallelContext, and in fact it was not 100% safe anyhow. fixed in the updated version, instead of checking the ParallelContext, the compute function is now checking if the output is allowed.